templates/kk/date/start.html

```diff
@@ -117,14 +117,77 @@ <h3 class="mt-3">Samarbetspartners</h3>
                                 {%for ad in ads%}
                                 <li class="company-logo text-center">
                                     <a href="{{ ad.company_url }}">
-                                        <img src="{{ ad.ad_url }}" alt="img">
+                                        <img src="{{ ad.ad_url }}" alt="img" onerror="this.src='/static/core/images/fallback.png'">
                                     </a>
                                 </li>
                                 {% endfor %}
                             </ul>
                         </div>
                     </div>
                 </div>
+                <div class="mobile-events-container">
```

Comment:
Probably should see if we can fix this without code duplication. Maybe create a new issue to fix it later.